scripts/run_app.sh

```diff
@@ -9,6 +9,9 @@ fi
 export DM_SUPPLIER_FRONTEND_API_AUTH_TOKEN=${DM_SUPPLIER_FRONTEND_API_AUTH_TOKEN:=myToken}
 export DM_API_URL=${DM_API_URL:=http://localhost:5000}
 export DM_API_AUTH_TOKEN=${DM_API_AUTH_TOKEN:=myToken}
+export DM_MANDRILL_API_KEY=${DM_MANDRILL_API_KEY:=not_a_real_key}
```

Comment:
Do we need to add the new env vars to the digitalmarketplace-aws stacks.yml before merging this or will this still work without them?